examples/nvflare/v2.0/hello-pytorch/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM python:3.8-slim-bullseye
```

Comment:
I think it might be over-complicating things to split the nvflare examples into different subdirectories by version.
Most of the examples aren't all that different between different versions, and we can explain in the README.md of each example which version it was built for.
WDYT?